website/docs/llm_configuration.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "custom_edit_url: https://github.com/microsoft/autogen/edit/main/website/docs/llm_configuration.ipynb\n",
+    "---\n",
+    "\n",
+    "# LLM Configuration\n",
+    "\n",
+    "In AutoGen, LLM's are a key component required for agents to understand and react. Each agent's access to and configuration of LLM's is defined by the `llm_config` argument in its constructor.\n",
+    "\n",
+    "The simplest possible LLM configuration that is configured to use `gpt-4` from OpenAI would be:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "llm_config = (\n",
```

Comment:
We could format the json specification of config list a bit better (use new lines and indentation) so that object spec is more readable?

Reply:
This is how black decides to format it

Reply:
damn I don't wanna mess with that. Ignore this for now then.